bip-update-process.md

```diff
@@ -0,0 +1,595 @@
+  Status: <Draft | Preliminary | Active | Abandoned>
+  Type: <Specification | Informational | Process>
+  Created: <date the BIP was assigned a number, in ISO 8601 (yyyy-mm-dd) format>
+  License: <abbreviation for approved license(s)>
+* License-Code: <abbreviation for code under different approved license(s)>
+* Discussion: <list of mailing list threads, or other noteworthy discussions in "date: link" format>
+* Requires: <BIP number(s)>
+* Replaces: <BIP number>
+* Superseded-By: <BIP number>
+```
+
+__BIP__: BIP number, or "?" before a number was assigned. The proposal must not self-assign a BIP number. Numbers are
+assigned by BIP Editors.
+
+__Layer__: The Layer header documents which layer of Bitcoin the BIP applies to. See [BIP 123](bip-0123.mediawiki) for
+definitions of the various BIP layers.
+
+__Authors__: The authors header lists the names and email addresses of the current proponents of the BIP. This may or
+may not be the original authors of the BIP. The format of each authors header value must be
+
+    Random J. User <address@dom.ain>
+
+If there are multiple authors, each should be on a separate line following [RFC
+2822](https://datatracker.ietf.org/doc/html/rfc2822.html) continuation line conventions.
+
+__Type__: The Type header specifies the type of BIP: Specification, Informational, or Process.
+
+__Created__: The Created header records the date that the BIP was assigned a number.
+
+__Discussion__: Discussion is used to record when new versions of the BIP are posted to bitcoin mailing lists. Dates
+should be in yyyy-mm-dd format, e.g. 2001-08-14. Discussion is permitted to be a link to a specific thread in a
+mailing list archive.
+
+__Requires__: BIPs may have a Requires header, indicating the BIP numbers that this BIP depends on.
+
+__Replaces/Superseded-By__: BIPs may have a Superseded-By header indicating that a BIP has been rendered obsolete by a
+later document; the value is the number of the BIP that replaces the current document. The newer BIP must have a
+Replaces header containing the number of the BIP that it rendered obsolete.
+
+#### Auxiliary Files
+
+BIPs may include auxiliary files such as diagrams. Auxiliary files must be included in a subdirectory for that BIP named
+`bip-XXXX`, where "XXXX" is the BIP number. File names in the subdirectory do not need to adhere to a specific
+convention.
+
+### BIP types
+
+* A **Specification BIP** defines a set of rules to enable a new feature or protocol change that affects the
+  interoperability of applications using Bitcoin. The distinguishing feature of a Specification BIP is that it can be
+  implemented, and applications using Bitcoin can be compliant with it. Specification BIPs must contain a specification
+  section as well as contain or link to a reference implementation.
+* An **Informational BIP** describes a Bitcoin design issue, provides general guidelines, or other information to the
+  Bitcoin community, but does not propose a new feature.
```

Comment:
Let me try to rewrite these entirely:

```suggestion
* A **Specification BIP** defines a set of technical rules affecting the
  interoperability of implementations. The distinguishing feature of a Specification BIP is that it can be
  implemented, and implementations can be compliant with it. Specification BIPs should come with or refer to a reference implementation.
* An **Informational BIP** describes a design issue, provides general guidelines, or other information to the
  Bitcoin community.
```